qiskit/algorithms/optimizers/gsls.py

```diff
@@ -25,6 +25,11 @@ class GSLS(Optimizer):
     An implementation of the line search algorithm described in
     https://arxiv.org/pdf/1905.01332.pdf, using gradient approximation
     based on Gaussian-smoothed samples on a sphere.
+
+    
+    Note: This component has some function that is normally random. If you want to reproduce behavior
```

Comment:
Sphinx comment. How about using 
`.. note::`

See the following link for more details. https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html